src/charm.py

```diff
@@ -389,27 +387,6 @@ def _write_vector_config(self, _):
                 if endpoint := relation.data[unit].get("endpoint", ""):
                     loki_endpoints.append(json.loads(endpoint)["url"])
 
-        if loki_endpoints:
```

Comment:
Removing this seems like the right thing to do.
Any chance there are metrics we could have a "self monitoring" alert in case loki is unreachable?